modules/gcp_bigquery_table/inputs.tf

```diff
@@ -28,3 +28,11 @@ variable "customer_managed_key_id" {
   type        = string
   description = "Describes the Cloud KMS encryption key that will be used to protect destination BigQuery table. The BigQuery Service Account associated with your project requires access to this encryption key."
 }
+
+variable "materialized_view" {
+  type = set(object({
```

Comment:
Does it support more than one object?

Reply:
Needed to use `set` because of the dynamic field in resource

Reply:
Can you add condition in variable to force length to 1?